proj_eclipse/src/main/java/smartmag/models/BoxModel.java

```diff
@@ -168,7 +169,7 @@ record = null;
 	 * 
 	 * @param quantita numero di unita di prodotto aggiunte
 	 */
-	protected void rifornisci(int quantita) {
+	public void rifornisci(int quantita) {
```

Comment:
Solo i modelli (MovimenModel) dovrebbe modificarla
```suggestion
	protected void rifornisci(int quantita) {
```